src/app/containers/Signup.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const Signup = () => {
+  return (
+    <div>
+      <h3>Sign up</h3>
+      <div>
+        <form method="POST" action="">
+          <input type="text" name="email" placeholder="email" />
+          <input type="password" name="password" placeholder="password" />
+          <input type="text" name="confirm" placeholder="Confirm password" />
+          <input type="submit" value="Sign up" />
```

Comment:
you wont need this to be a input of type submit, all the form submission will be handled by react, just change it to a button